graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlWebSocketService.java

```diff
@@ -74,4 +82,9 @@ public WebSocket handle(ServiceRequestContext ctx, WebSocket in) {
         in.subscribe(new GraphqlWebSocketSubscriber(protocol, outgoing));
         return outgoing;
     }
+
+    @Override
+    public HttpServiceOptions options() {
```

Comment:
Question) Is it possible to just override this method from `WebSocketService` so that other websocket based services can also take advantage of the default options by default?

e.g.
```
interface WebSocketService {
  default HttpServiceOptions options() {
    return DEFAULT_OPTIONS;
  }
}

...


class WebSocketServiceBuilder {
  static final HttpServiceOptions DEFAULT_OPTIONS = ...
}
```

Reply:
I see, isn't the point of this PR to apply default options to `WebSocketService` though?
Let's wait for input from the original author of the test (@ikhoon ) then 😄

Reply:
Let's remove `DelegatingWebSocketServiceTest#shouldNotSetDefaultSettings()`. `DelegatingWebSocketService` can handle both WebSocket requests and HTTP requests. The default WebSocket timeout settings didn't make sense for `DelegatingWebSocketService`.

`WebSocketService.option()` is an interface method whose implementations may override the default value behavior.

Reply:
Let me work on this comment myself.